sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/csv/CSVOptions.scala

```diff
@@ -166,8 +166,6 @@ class CSVOptions(
 
   val quoteAll = getBool("quoteAll", false)
 
-  val inputBufferSize = 128
```

Comment:
The default is `1024*1024 characters`. I just wonder how much we will increase memory consumption in this way.

Reply:
This controls the buffer size in the reader (https://github.com/uniVocity/univocity-parsers/blob/master/src/main/java/com/univocity/parsers/common/input/DefaultCharInputReader.java). So I think it will increase from 128 bytes to 1 MB which shouldn't be a big deal.